examples/vlm_ptq/README.md

```diff
@@ -32,7 +32,7 @@ Please refer to the [llm_ptq/README.md](../llm_ptq/README.md#getting-started) fo
 
 ### Current out of the box configs
 
-Please refer to the [llm_ptq/README.md](../llm_ptq/README.md#current-out-of-the-box-configs) for details on the current out of the box configs.
+Please refer to the [llm_ptq/README.md](../llm_ptq/README.md#support-matrix) for details on the current out of the box configs.
```

Comment:
We already have a support matrix for VLM right after this so seems redundand?
Perhaps we remove this section?